edu.cuny.hunter.mylyngit.core/src/edu/cuny/hunter/mylyngit/core/utils/Vertex.java

```diff
@@ -56,4 +56,12 @@ public String getFile() {
 	public String getMethod() {
 		return method;
 	}
+
+	public int getCommit() {
+		return commit;
+	}
+
+	public void setCommit(int commit) {
+		this.commit = commit;
+	}
```

Comment:
What warning does this fix?

Reply:
The value of the field Vertex.commit is not used

Reply:
Should it be removed then?

Reply:
Seems that these warnings are all being suppressed rather than being dealt with.

Reply:
I am not sure whether we need this information in the future. I could remove it.

Reply:
I dealt with some warnings. For the warnings I could not process, I suppressed them.

Reply:
OK just leave it then.